.github/dependabot.yml

```diff
@@ -1,16 +1,18 @@
   version: 2
   updates:
-    - directory: "/"
+    - package-ecosystem: "pip"
+      directory: "/"
       schedule:
         interval: "weekly"
         day: "monday"
         time: "08:00"
         timezone: "America/Los_Angeles"
-      package-ecosystem: "pip"
-      reviewers:
```

Comment:
I don't believe reviewers should be necessary, as we have [CODEOWNERS](https://github.com/Netflix/lemur/blob/master/.github/CODEOWNERS) set up.